app/models/anime.rb

```diff
@@ -59,6 +59,7 @@ class Anime < ApplicationRecord
   has_many :anime_productions, dependent: :destroy
   has_many :anime_characters, dependent: :destroy
   has_many :anime_staff, dependent: :destroy
+  has_many :media_attribute, through: :anime_media_attributes
```

Comment:
Don't you need to also declare `has_many :anime_media_attributes` for this to work?  Also do we even need an association to `media_attribute`, it's just a table with like... one row?